grpcgcp/gcp_balancer.go

```diff
@@ -393,6 +416,29 @@ func (gb *gcpBalancer) UpdateSubConnState(sc balancer.SubConn, scs balancer.SubC
 	gb.mu.Lock()
 	defer gb.mu.Unlock()
 	s := scs.ConnectivityState
+
+	if scRef, found := gb.refreshingScRefs[sc]; found {
```

Comment:
Have you considered the interaction of unresponsive connection feature and fallback. If a connection is unresponsive ideally it should not be a fallback as well.